docs/advanced-topics/dockstore-tools-overhaul.rst

```diff
@@ -11,24 +11,25 @@ This documentation explains:
 Dockstore Tools Prior to 1.12
 -----------------------------
 
-In a very broad sense, we defined a tool on Dockstore as a CWL or WDL program that performs a single task in a unique Docker container. Dockstore tool versions were based on the image's tags and we required
-that the Docker image be owned by the Dockstore user. However, working on GitHub App registration for tools caused a lot of internal discussion because we realized that Dockstore tool support was honestly confusing.
+In a very broad sense, we defined a tool on Dockstore as a CWL or WDL program that performs a single task in a unique Docker container. Dockstore tool versions were based on the Docker image's tags and we required
+that the image be owned by the Dockstore user. However, working on GitHub App registration for tools caused a lot of internal discussion because we realized that Dockstore tool support was confusing.
 
-For starters, we created distinctions between tools and workflows even when one did not exist in the language's own specification.
+For starters, we created distinctions between tools and workflows even when one did not exist in one language's specification.
 When Dockstore was created, CWL was the first descriptor language we supported. CWL does have a very clear distinction between a tool and a workflow and we followed it.
```

Comment:
Did we, though? A CWL tool does not require a Docker image, so our definition didn't quite follow it, even though it did follow CommandLineTool vs Workflow.

Reply:
Good point!